py/demo/dashboard_grey.py

```diff
@@ -15,7 +15,7 @@ async def show_grey_dashboard(q: Q):
                     ui.zone('top', direction=ui.ZoneDirection.ROW, size='25%'),
                     ui.zone('middle', direction=ui.ZoneDirection.ROW, size='25%'),
                     ui.zone('middle2', direction=ui.ZoneDirection.ROW, size='25%'),
-                    ui.zone('bottom', direction=ui.ZoneDirection.ROW, size='25%'),
+                    ui.zone('bottom', direction=ui.ZoneDirection.ROW, size='20%'),
```

Comment:
This does not address the problem why the footer overlaps the body, regardless of the height of the body. For example, the problem persists even if you change the body size to `800px`.

Reply:
So we have to revert this to 25%, and instead investigate what's wrong with the layout system.